cmake/utils/calculatesha1.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <fstream>
+#include <string>
+
+#include <libcellml/generatorprofile.h>
+
+#include "generatorprofiletools.h"
+#include "utilities.h"
+
+int main()
+{
+    auto cProfile = libcellml::GeneratorProfile::create();
```

Comment:
Might want to rename that variable to `cGeneratorProfile`. Also have `pyGeneratorProfile`, `cGeneratorProfileString`, and `pyGeneratorProfileString`.

Reply:
[86a9d00](https://github.com/cellml/libcellml/pull/1054/commits/86a9d006aa0d4cd9108a57e6ef42ce7652abd9f7)